_posts/2021-09-13-user-defined-value-types.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+published: true
+title:  User Defined Value Types in Solidity
+date:   2021-09-13
+author: Solidity Team
+category: Explainers
+---
+
+Solidity [Solidity v0.8.8](https://github.com/ethereum/solidity/releases/tag/v0.8.8) introduces
+[user defined value
+types](https://docs.soliditylang.org/en/develop/types.html?#user-defined-value-types) as a means to
+create zero-cost abstractions over an elementary value type.
```

Comment:
What is the benefit of a zero-cost abstraction? (maybe mention additional type-safety, etc)